### PR TITLE
Fix assets without name causing analyze failure

### PR DESCRIPTION
**Pull request by ZKHelloworld**
Fix this issue #385

### CODE REVIEW

**Comment by valscion** [comment]
Can you add a test case and a changelog entry?

**Comment by ZKHelloworld** [comment]
@valscion 
sure.

**Comment by ZKHelloworld** [comment]
@valscion 
Done.

**Comment by valscion** [comment]
Yeah this looks good to me — would @th0r want to take a look as you originally asked for a new PR here? https://github.com/webpack-contrib/webpack-bundle-analyzer/issues/385#issuecomment-707007310

File: test/stats/with-non-asset-asset/a-clone.js [hunk 1]
```diff
@@ -0,0 +1 @@
+module.exports = 'module a';
```
Comment (th0r) [inline test/stats/with-non-asset-asset/a-clone.js:1]: Why do we need this file?

**Review by th0r — CHANGES_REQUESTED** [review]

File: test/stats/with-non-asset-asset/a-clone.js (hunk 1 above)
Comment (ZKHelloworld) [reply test/stats/with-non-asset-asset/a-clone.js:1]: Not necessary.

Using `CopyWebpackPlugin` could cause `hidden assets` added in stats.json.

 This file is a copied asset generated by a test webpack build.

I can delete this file if it's wanted.

Comment (th0r) [reply test/stats/with-non-asset-asset/a-clone.js:1]: Yep, delete it please.

Comment (ZKHelloworld) [reply test/stats/with-non-asset-asset/a-clone.js]: Done

**Comment by th0r** [comment]
Could you also merge latest `master` into this branch please? `CHANGELOG.md` diff looks quite strange now.

**Comment by ZKHelloworld** [comment]
> Could you also merge latest `master` into this branch please? `CHANGELOG.md` diff looks quite strange now.

Done.

**Review by th0r — APPROVED** [review]

**Comment by th0r** [comment]
Thanks!

**Comment by valscion** [comment]
Released in v4.2.0 :relaxed:. Thank you for your contributions!

**Pull request merged**